src/Sensor/SensorEcu.cpp

```diff
@@ -9,30 +9,28 @@
 #define ECU_DATA_FIELD_LENGTH   0x16
 #define ECU_SERVICE_ID          0x50
 
-SensorEcu::SensorEcu(USARTSerial *serial)
-{
+#define STALE_INTERVAL          2000
```

Comment:
Just a thought, maybe this should be common across all sensors?

Reply:
I've set different times for Can sensors, but 2 seconds seems like a reasonable interval

Reply:
Yeah I think it could be common across all sensors!

Reply:
I can throw this in then.  Maybe put the macro definition in Sensor.h?